src/components/Button/Button.stories.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-anonymous-default-export */
+import React from 'react';
+import EmailButton from './StyledButton';
+
+export default {
+	title: 'Components/Button',
+	component: EmailButton,
+	decorators: [
+		Story => {
+			return (
+				<div style={{ padding: '3em', maxWidth: '400px', position: 'relative' }}>
+					<Story />
+				</div>
+			);
+		},
+	],
+};
+
+const props = { props: { email: 'hallo@hallo.de' } };
```

Comment:
If I did understand correclty, then is `EmailButton` a styled component where you don not work with anything `props` related yet. Do you really need `const props` for your `Story`? 🤔